### ISSUE TITLE
Is there a way to set all IDs as integers instead of strings?

### ISSUE THREAD

**Issue by kenearley** [miragejs/server#133]
I've tried to write a custom IdentityManager to use integers instead of strings for all ids. However, it seem to force conversion to strings in a few places, [here](https://github.com/miragejs/server/blob/master/lib/db-collection.js#L409) and [here](https://github.com/miragejs/server/blob/master/lib/db-collection.js#L429). Am I missing something?

**Comment by kenearley** [comment]
I have attempted to make a PR to allow for integers. However, there were too many tests that failed and they seem to be [explicitly testing that integer ids are converted to strings](https://github.com/miragejs/server/blob/686cd86ad6fddb3117920cbdaba35da006dcaa1e/__tests__/external/shared/serializers/base/basic-test.js#L53-L64).

**Comment by samselikoff** [comment]
Here's some background on the issue: https://github.com/samselikoff/ember-cli-mirage/issues/67

I believe when we didn't coerce these, there were lots of issues where `schema.user.find(1)` or `schema.user.find('1')` wouldn't work.

But you should absolutely be able to do what you want. Question for you: what's leading to you wanting to use ints? Does your production API send ints over in the JSON payloads?

**Comment by kenearley** [comment]
> Here's some background on the issue: samselikoff/ember-cli-mirage#67
Thanks. I'll read up on it.

> what's leading to you wanting to use ints?
Mostly because that's what our production server is sending, but for the UI specifically, it's being used for sorting purposes, which I will admit probably isn't the most compelling reason, but we are displaying tables with an id column which we sort on.

> Does your production API send ints over in the JSON payloads?
Yes.

I've been able to make it work as it is, for now. I assume I'll just make the conversion in my custom serializer to handle it.

**Comment by samselikoff** [comment]
Yes, I think the appropriate place for this is the serializer later. Mirage’s database already has several conventions for its ORM to work. Mirages database data doesn’t need to match your product system - it’s really just about getting the API responses to match your product responses. 

Let me know if the code in your serializer was annoying to implement!
